sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusProcessorTest.java

```diff
@@ -485,7 +486,7 @@ private ServiceBusClientBuilder.ServiceBusReceiverClientBuilder getBuilder(
 
         ServiceBusReceiverAsyncClient asyncClient = mock(ServiceBusReceiverAsyncClient.class);
         when(receiverBuilder.buildAsyncClient()).thenReturn(asyncClient);
-        when(asyncClient.receiveMessagesWithContext()).thenReturn(messageFlux);
+        when(asyncClient.receiveMessagesWithContext()).thenReturn(messageFlux.publishOn(Schedulers.boundedElastic()));
```

Comment:
Mimic Low-Level Receiver's behavior of using `Schedulers.boundedElastic` when publishing, this also ensures the same threading test setup as earlier, i.e., `runOn` used to set the `Schedulers.boundedElastic` for any max-current-call value.